tests/unit/backend/layers/business/test_business.py

```diff
@@ -1398,18 +1398,34 @@ def test_delete_collection_version_ok(self):
         self.assertEqual(version.version_id, published_collection.version_id)
         self.assertNotEqual(version.version_id, new_version.version_id)
 
-    def test_tombstone_collection_ok(self):
+    def test_tombstone_then_resurrect_collection_ok(self):
```

Comment:
this test replacement improves upon the original `test_tombstone_collection_ok` and also includes resurrection test coverage.